sugar/lib/src/time/zone/providers/universal/universal_provider.dart

```diff
@@ -0,0 +1,707 @@
+// ignore_for_file: public_member_api_docs, sort_constructors_first
```

Comment:
I think this file needs to be broken down into several smaller files, it's difficult to follow what's happening.

Reply:
Sure

Reply:
I'll refactor it to make it much easier to understand what going on.